examples/pico/interceptors/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.applications</groupId>
+        <artifactId>helidon-nima</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../../../applications/nima/pom.xml</relativePath>
+    </parent>
+    <groupId>io.helidon.examples.pico</groupId>
+    <artifactId>helidon-examples-pico-interceptors</artifactId>
+    <name>Helidon Pico Examples Interceptors</name>
+
+    <description>
+        Example usages of Pico Interceptors.
+    </description>
+
+    <properties>
+        <mainClass>io.helidon.examples.pico.interceptors.Main</mainClass>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.helidon.examples.pico</groupId>
+            <artifactId>helidon-examples-pico-basics</artifactId>
+            <version>${helidon.version}</version>
```

Comment:
Same as comment above about the version. If possible, should be fixed everywhere.